howto/getting-started/deploy-workload-manager.md

```diff
@@ -0,0 +1,163 @@
+---
+relatedlinks: "[Get&#32;started&#32;with&#32;LXD](https://documentation.ubuntu.com/lxd/en/latest/tutorial/first_steps/), [Get&#32;started&#32;with&#32;Juju](https://juju.is/docs/juju/tutorial), [Slurm&#32;website](https://slurm.schedmd.com/overview.html)"
+---
+
+# Deploy the workload manager
+
+An HPC workload manager is the base component of your Charmed HPC cluster.
+All the applications and auxiliary services that can be deployed with Charmed HPC
+are integrated with the workload manager to compose the functionality of your cluster.
+
+[Slurm](https://slurm.schedmd.com/overview.html) is currently the only
+supported workload manager implementation for Charmed HPC.
+
+## Prerequisites
+
+To successfully deploy the workload manager of your Charmed HPC cluster, you
+will at least need:
+
+- A machine running a [currently supported Ubuntu LTS version](https://ubuntu.com/about/release-cycle).
+- An initialised [LXD](https://canonical.com/lxd) instance.
+- A [Juju](https://juju.is) client.
+
+## Getting started
+
+Now, before you can deploy the workload manager of your cluster, you need to:
+
+1. Initialise the machine cloud that will provide the instances that the workload
+   manager's services will run within.
+2. Create the model that will hold the workload manager's services.
```

Comment:
How-to's completely skip describing the next steps that the user will do, only putting the steps directly.

Reply:
Sounds good. I'm going to remove the whole "Getting started" section. I think it's better to just have the "Initialise the machine cloud" and "Create a model for the cluster" sections just be top-level sections rather than nested under "Getting started"